packages/babel/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "rollup-plugin-babel",
+  "version": "5.0.0-alpha.1",
+  "description": "Seamless integration between Rollup and Babel.",
+  "license": "MIT",
+  "repository": "rollup/rollup-plugin-babel",
+  "author": "Rich Harris",
+  "homepage": "https://github.com/rollup/rollup-plugin-babel",
+  "bugs": {
+    "url": "https://github.com/rollup/rollup-plugin-babel/issues"
+  },
+  "main": "./dist/rollup-plugin-babel.cjs.js",
+  "engines": {
```

Comment:
I'd like to avoid source transpilation (apart of producing CJS from ESM source) which shouldn't be needed for node 10 which will become the oldest LTS in a few days. WDYT about this?

Reply:
The source is ESM, so rollup should produce both CJS and ESM. bear in mind that the plugins need to follow Rollup's lead on which engines are supported. As that's a breaking change, I doubt @lukastaegert will want to rush to that, even if a version has fallen out of LTS. It's probably going to take some time, and that build step doesn't really cost us anything.

Reply:
What are currently supported engines then? Hard to find that info quickly in Rollup's repo.

Reply:
What are currently supported engines then? Hard to find that info quickly in Rollup's repo.

Reply:
Node version (v8.0.0+) and Rollup v1.20.0+. You can check the README of individual plugins and its listed under "Requirements". It might be worthwhile to add it to the root README somewhere.

Reply:
@Andarist agreed. when Node 8 exits LTS my plan is to introduce `engines` and a requirements section to rollup core. that will require a new major, so it's something that I/we need to get lukas on board with. For the time being, all plugins (and Rollup, even though it isn't documented) are Node 8+.

Reply:
Ok, I've added transpilation using itself.